app/services/stats/compute_follow_up_seen_rate.rb

```diff
@@ -0,0 +1,66 @@
+module Stats
+  class ComputeFollowUpSeenRate < BaseService
+    def initialize(follow_ups:, target_delay_days:, consider_orientation_rdv: false)
+      @follow_ups = follow_ups
+      @target_delay_days = target_delay_days
+      @consider_orientation_rdv = consider_orientation_rdv
+    end
+
+    def call
+      result.value = compute_percentage_seen_within_delay
+    end
+
+    private
+
+    attr_reader :follow_ups, :target_delay_days, :consider_orientation_rdv
+
+    def compute_percentage_seen_within_delay
+      (follow_ups_seen_within_delay.count / (mature_follow_ups.count.nonzero? || 1).to_f) * 100
+    end
+
+    def follow_ups_seen_within_delay
+      @follow_ups_seen_within_delay ||= begin
+        matches = []
+        mature_seen_follow_ups.find_in_batches(batch_size: 1000) do |batch|
+          matches += batch.select do |follow_up|
+            meets_delay_criteria?(follow_up)
+          end
+        end
+        matches
+      end
+    end
+
+    def meets_delay_criteria?(follow_up)
+      return false unless follow_up.days_between_follow_up_creation_and_first_seen_rdv
```

Comment:
nitpick: peut-être un peu plus lisible si on fait
```suggestion
      return false unless follow_up.seen_rdvs?
```